rtf_vis_tool/config-overrides.js

```diff
@@ -0,0 +1,28 @@
+const webpack = require('webpack');
```

Comment:
@stepanyanhayk thanks for the PR. Could you add a file global comment describing the purpose and contents of this file, and add inline comments about why the code is needed?